vendor/engines/sanger_sequencing/app/models/sanger_sequencing/submission.rb

```diff
@@ -22,9 +22,15 @@ class Submission < ActiveRecord::Base
     alias purchased_at ordered_at
 
     scope :purchased, -> { joins(:order).merge(Order.purchased.order(ordered_at: :desc)) }
-    scope :ready_for_batch, -> { purchased.merge(OrderDetail.new_or_inprocess).where(batch_id: nil) }
     scope :for_facility, ->(facility) { where(orders: { facility_id: facility.id }) }
 
+    BATCHABLE_STATES = %w(new inprocess complete).freeze
```

Comment:
Ah okay, so this makes sense if a submission gets completed accidentally without getting attached to a batch. What if the submission is run as part of a batch, but for some reason or other needs to be re-run? If I'm understanding the story correctly, that's in scope...